contracts/other/notional/NotionalMultiOracle.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import "@yield-protocol/utils-v2/contracts/access/AccessControl.sol";
+import "@yield-protocol/utils-v2/contracts/cast/CastBytes32Bytes6.sol";
+import "@yield-protocol/utils-v2/contracts/token/IERC20Metadata.sol";
+import "@yield-protocol/vault-interfaces/IOracle.sol";
+
+
+/**
+ * @title NotionalMultiOracle
+ * @notice We value fCash assets at face value
+ */
+contract NotionalMultiOracle is IOracle, AccessControl {
+    using CastBytes32Bytes6 for bytes32;
+
+    event SourceSet(bytes6 indexed notionalId, bytes6 indexed underlyingId, address underlying);
+
+    struct Source {
+        uint8 baseDecimals;
+        uint8 quoteDecimals;
+        bool inverse;
+    }
+
+    mapping(bytes6 => mapping(bytes6 => Source)) public sources;
+
+    /// @dev Set or reset an oracle source and its inverse
+    function setSource(bytes6 notionalId, bytes6 underlyingId, IERC20Metadata underlying)
+        external auth
+    {
+        require (notionalId != underlyingId, "Wrong input");
+        sources[notionalId][underlyingId] = Source({
+            baseDecimals: 18, // I'm assuming here that fCash has 18 decimals
```

Comment:
is there no way to confirm this?

Reply:
In my TODO for this morning

Reply:
![image](https://user-images.githubusercontent.com/38806121/155493218-4483a794-defe-4927-8a5b-2c0ef59d233d.png)

Reply:
Upon testing it's 8 decimals for everything.